packages/a11y-testing/tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "composite": true,
```

Comment:
had to removed it, because import was not working properly, thanks to @layershifter for help :)

Reply:
but then I see :(
build is failing because of it:
verb @fluentui/react-northstar build |  tsconfig.json(18,5): error TS6306: Referenced project '/mnt/work/6/s/packages/a11y-testing' must have setting "composite": true.

Please @layershifter  I would need to ask you for help again :)